en/common/command-line.mdown

```diff
@@ -358,25 +369,161 @@ You can also use `parse jssdk` to check which Parse JavaScript SDK version is cu
 
 ```bash
 $ parse jssdk
-Current JavaScript SDK version is 1.2.13
+Current JavaScript SDK version is 1.5.0
 $ parse jssdk -a
-  1.2.18
-  1.2.17
-  1.2.16
-  1.2.15
-  1.2.14
-* 1.2.13
-  1.2.12
-  1.2.11
-$ parse jssdk 1.2.18
-Current JavaScript SDK version is 1.2.18
+*  1.5.0
+   1.4.2
+   1.4.0
+   1.3.5
+   1.3.4
+   1.3.3
+   1.3.2
+$ parse jssdk 1.4.2
+Current JavaScript SDK version is 1.4.2
 $ parse jssdk -a
-* 1.2.18
-  1.2.17
-  1.2.16
-  1.2.15
-  1.2.14
-  1.2.13
-  1.2.12
-  1.2.11
+   1.5.0
+*  1.4.2
+   1.4.0
+   1.3.5
+   1.3.4
+   1.3.3
+   1.3.2
+```
+
+## Updating
+
+You can update the command line tool using `parse update`. 
+It will update your Parse command line tool to the latest version.
+
+For instance, if you run this command on Mac OS:
+
+```bash
+Downloading binary from https://github.com/ParsePlatform/parse-cli/releases/download/release_${LATEST_VERSION}/parse.
+Successfully updated binary at: /usr/local/bin/parse
 ```
+
+## Secure Config Format
+
+If you are using a version of command line tool older than 2.2.2 the cloud code project you create looks like so:
```

Comment:
"If you are using a version of the command line tool older than 2.2.2, the Cloud Code project you create looks like so:"

The last fragment of that sentence still sounds a bit odd to me though...